app/views/application/tips.html.erb

```diff
@@ -0,0 +1,85 @@
+<main>
+  <%= render "header" %>
+</main>
+<center>
+<h2 style="padding:10px">
+How to Bird
+</h2>
+</center>
+<p style="padding:80px; text-indent:50px">
+
+Location, time of day, weather, and time of year can influence what types and numbers of birds you might observe.
+
+Get to know your regulars. Some birds in your neighborhood are common, year-round residents. By learning to identify these birds by sight and sound, and by recognizing their habits (How do they move? What do they eat? What do they sound like?) you will better be able to notice when something unusual comes along.
+
+Find a good spot. You might go for a drive through a wildlife area, walk on a trail, or sit in one spot and wait for the birds to come to you. One key is to find a place where two habitats meet, such as the edge of a forest and a meadow, or where muddy shorelines meet the water. Finding a spot where birds can find food and water can increase your chances of finding interesting species.
```

Comment:
(Opinion!) Putting strong tags around the first line of each paragraph might be nice effect.